From 438cac05e445bbe8ef5937f25d26d99591d659dc Mon Sep 17 00:00:00 2001
From: undef <gitlab@undef.tools>
Date: Sat, 23 Apr 2022 02:52:20 +0000
Subject: [PATCH] Add evdev rule for ppkb model

This allows the keyboard model to be picked up automatically on wayland
based systems.

Backported by elagost from:
https://gitlab.freedesktop.org/xkeyboard-config/xkeyboard-config/-/merge_requests/325
---
 rules/0026-evdev.m_s.part | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rules/0026-evdev.m_s.part b/rules/0026-evdev.m_s.part
index c79a02e2..2dd72051 100644
--- a/rules/0026-evdev.m_s.part
+++ b/rules/0026-evdev.m_s.part
@@ -1,5 +1,6 @@
 ! model		=	symbols
   $evdevkbds    =   +inet(evdev)+inet(%m)
   chromebook    =   +inet(evdev)+inet(chromebook)
+  ppkb          =   +inet(evdev)+inet(ppkb)
   applealu_jis  =   +inet(evdev)+macintosh_vndr/jp(alujiskeys)
   *             =   +inet(evdev)
-- 
